### PR TITLE
Addbiz nodes

### PR DESCRIPTION
**Pull request by alajko**
New FN.

### CODE REVIEW

File: core/resharding.go [hunk 1]
```diff
@@ -27,7 +27,7 @@ const (
 	// GenesisShardSize is the size of each shard at genesis
 	GenesisShardSize = 100
 	// GenesisShardHarmonyNodes is the number of harmony node at each shard
-	GenesisShardHarmonyNodes = 75
+	GenesisShardHarmonyNodes = 74
```
Comment (rlan35) [inline core/resharding.go]: Why change to 74?

Comment (alajko) [reply core/resharding.go]: Discussed offline.

Comment (rlan35) [reply core/resharding.go]: Please change back to 75.

Comment (alajko) [reply core/resharding.go]: Dicussed offline, Please approve.

**Pull request merged**